index.html

```diff
@@ -328,7 +328,10 @@ <h2 class="section-title">Sponsors</h2>
 				<h3>Silver Level Sponsors</h3>
 	      <a href="http://cdk-global.com" target="_blank"><img src="assets/img/logos/cdk-logo.png" /></a>
 	      <p><a href="http://cdk-global.com" target="_blank">CDK Global</a> is the largest global provider of integrated information technology and digital marketing solutions to the automotive retail industry.</p>
-	    </div>
+	    	<br/>
```

Comment:
Instead of placing the kcura logo in the same column underneath the cdk logo, I think we should give it it's own column in the grid. The grid is vertically aligned so making that column longer by adding the second logo stretches out the prospectus blurb column. Happy to help if needed :)

![screen shot 2017-04-04 at 6 45 00 pm](https://cloud.githubusercontent.com/assets/11186572/24762142/759288ec-1abb-11e7-9b66-21aa869351a7.png)